ansible_navigator/actions/collections.py

```diff
@@ -364,6 +367,7 @@ def _parse(self, output) -> None:
                 self._logger.info("[catalog_collections]: %s", msg)
 
         self._logger.debug("catalog collections scan path: %s", parsed["collection_scan_paths"])
+        self._collection_scanned_paths = sorted(parsed["collection_scan_paths"].split(":"))
```

Comment:
Should the scan path be sorted? I think the order of the scan path should be preserved while showing it in UI.

Reply:
that's a good point, i sorted it so it "looked" good, will fix